app/views/organisations/_organisation_list.html.erb

```diff
@@ -0,0 +1,31 @@
+<section class="app-table-group" tabindex="0" aria-labelledby="<%= title.dasherize %>">
+  <table id="all-organisations-table" class="govuk-table">
```

Comment:
Any reason not to use the Table component from the gem here like the User's view does?